### PR TITLE
Update screenshots

### PR DESCRIPTION
**Pull request by paigecalvert**
https://deploy-preview-1511--replicated-docs.netlify.app/vendor/vendor-portal-application-settings
https://deploy-preview-1511--replicated-docs.netlify.app/vendor/licenses-verify-fields-sdk-api#use-your-public-key-to-verify-license-field-signatures

### CODE REVIEW

File: docs/vendor/vendor-portal-application-settings.md [hunk 1]
```diff
@@ -13,5 +13,5 @@ The following describes each of the application settings:
 - **Application name:** The application name is initially set when you first create the application in the vendor portal. You can change the name at any time so that it displays as a user-friendly name that your team can easily identify.
 - **Application slug:** The application slug is used with the replicated CLI and with some of the kots CLI commands. You can click on the link below the slug to toggle between the application ID number and the slug name. The application ID and application slug are unique identifiers that cannot be edited.
 - **Service Account Tokens:** Provides a link to the the **Service Accounts** page, where you can create or remove a service account. Service accounts are paired with API tokens and are used with the Vendor API to automate tasks. For more information, see [Using Vendor API Tokens](/reference/vendor-api-using).
-- **Replicated Kots:** Indicates if the application has a KOTS entitlement.
+- **Scheduler:** Displayed if the application has a KOTS entitlement.
```
Comment (paigecalvert) [inline docs/vendor/vendor-portal-application-settings.md:16]: @miaawong one question about the changes to the App Settings page: is the Scheduler section only displayed if the app has the KOTS entitlement?

Comment (miaawong) [reply docs/vendor/vendor-portal-application-settings.md:16]: I believe so. @alicenstar can you confirm?

Comment (alicenstar) [reply docs/vendor/vendor-portal-application-settings.md:16]: @paigecalvert Yes, that is correct! It won't be shown otherwise.

Comment (paigecalvert) [reply docs/vendor/vendor-portal-application-settings.md:16]: thank you!

**Pull request merged**